runtime/bf16_support.cu

```diff
@@ -88,4 +88,25 @@ __device__ __inline__ __bfloat operator|(const __bfloat x, const __bfloat y) {
   return val;
 }
 
+__device__ __inline__ bool __heq(const __bfloat a, const __bfloat b) {
```

Comment:
Moved from `casts.cu`, it was mistakenly moved from here to that file in a past PR.